OracleSOASuite/dockerfiles/12.2.1.3/README.md

```diff
@@ -36,15 +36,15 @@ If that returns a username (which is the first field), you can skip the below us
 
 Once the `oracle` user is created, run the following commands as a root user:
 
-    # mkdir -p /u01/DockerVolume/SOAVolume/
-    # chown 1000:1000 /u01/DockerVolume/SOAVolume/
-    # chmod 700 /u01/DockerVolume/SOAVolume/
+    # mkdir -p /u01/DockerVolume/SOAVolume/SOA
+    # chown -R 1000:1000 /u01/DockerVolume/SOAVolume/
```

Comment:
This doesn't take into account the scenario where the `getent` command returns a value that isn't `oracle` and the `gid` of the existing user isn't `1000`.  You probably don't need to set the `gid` here anyway, given the value of the `chmod` command next.